vault/backend.go

```diff
@@ -585,7 +632,7 @@ func GetNet(strNet string) (*chaincfg.Params, error) {
 		return &chaincfg.RegressionNetParams, nil
 
 	default:
-		return nil, errors.New("invalid network specified: " + strNet)
+		return nil, ErrInvalidNetwork
```

Comment:
From what I understand, the feature is upgraded/downgraded transparently/lazily when licensing/vault configuration changes, but allows enterprise users to have an extra layer of encryption if they enable it. By default, when not using an enterprise license, it has no effect.

When actually enabled, it can have the effect of:
 * slightly slower node creation/import
 * slightly slower first use of a node's secret, which is then cached locally by the vault instance "un-seal-wrapped (but still encrypted by Vault's cryptographic barrier)", per the link you mentioned
 * potential disruption in a non-HA setup if an HSM is used to seal the secret and the HSM becomes unavailable

Reply:
Okay, clear 👍